### PR TITLE
ENT-3199 | Added course mode in manual enrollment API.

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

File: ecommerce/extensions/api/v2/views/orders.py [hunk 1]
```diff
@@ -268,9 +274,8 @@ def _create_single_order(self, enrollment, request_user, request_site):
             return dict(enrollment, status=self.FAILURE, detail="Course not found", new_order_created=None)
 
         seat_product = course.seat_products.filter(
-            attributes__name='certificate_type'
-        ).exclude(
-            attribute_values__value_text='audit'
+            attributes__name='certificate_type',
+            attribute_values__value_text=mode
```
Comment (HammadAhmadWaqas) [inline ecommerce/extensions/api/v2/views/orders.py:278]: As manual enrollment is only for the paid modes I think we should make this sure here to not create orders for non `audit` enrollments.
https://github.com/edx/edx-enterprise/commit/defcb9458feddc67cd9e19cbc555e7db98648bee

@irfanuddinahmad what would do you suggest?

**Review by HammadAhmadWaqas — COMMENTED** [review]
just a comment, otherwise looks good to me,

File: ecommerce/extensions/api/v2/views/orders.py (hunk 1 above)
Comment (hasnain-naveed) [reply ecommerce/extensions/api/v2/views/orders.py:278]: @HammadAhmadWaqas in [enterprise](https://github.com/edx/edx-enterprise/blob/1ffe497e8b48fab58a75c8cb09223a1ba721a0ea/enterprise/admin/views.py#L819) we are calling for only paid modes so we don't need to add a check here.

Comment (HammadAhmadWaqas) [reply ecommerce/extensions/api/v2/views/orders.py:278]: @hasnain-naveed Yes, you are right, but in future, someone could call that API with audit mode which was not possible before(Aa we were excluding it.) and API would create an order for the Audit track.

**Review by HammadAhmadWaqas — APPROVED** [review]

**Review by sameenfatima78 — APPROVED** [review]
Just tested your branch...LGTM!

**Review by moconnell1453 — APPROVED** [review]

**Pull request merged**